### DISCUSSION TITLE
Alerting via Log in addition to Slack alerting feature

### DISCUSSION THREAD

**Discussion by olad32** [BerriAI/litellm#3497] [Q&A]
Hi, many recent commits adds alerting features via Slack, but i do not use Slack (and don't want to use it), is it possible to also have alerting via simple log (my log ingestion system will then alert me)
Thanks

**Comment by ishaan-jaff** [comment]
@olad32 the slack alerting simply sends it to a webhook 

- do you have a specific webhook you'd prefer sending it too ? 
- where would you like the alert to land ?

**Reply by ishaan-jaff** [reply]
bump @olad32 - where would you like the alerts ?

**Comment by olad32** [comment]
Hello, I don't want to send it to a webhook, I have a company wide log ingestion system that can already generate alerts based on logs emitted by applications. Alerts system based on logs are less intrusive for application performance, stability and maintenance, even if it adds a bit more latency.
So my needs is to be able to log the same information that is currently sent to slack by litellm. 


Le 11 mai 2024 23:44:40 GMT+02:00, Ishaan Jaff ***@***.***> a écrit :
